### PR TITLE
Bump master to 4.6.0

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
||

## Description

This PR changes the version in the `master` branch to 4.6.0.

### CODE REVIEW

**Review by verdx — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]

File: rpms/SPECS/wazuh-manager.spec [hunk 1]
```diff
@@ -843,6 +843,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Mon Sep 01 2023 support <info@wazuh.com> - 4.6.0
```
Comment (okynos) [inline rpms/SPECS/wazuh-manager.spec:846]: Sep 01 is Friday. Fix it please.

**Review by okynos — CHANGES_REQUESTED** [review]
Minor fix, GJ.

File: rpms/SPECS/wazuh-manager.spec (hunk 1 above)
Comment (DFolchA) [reply rpms/SPECS/wazuh-manager.spec]: Done [2140283](https://github.com/wazuh/wazuh-packages/pull/2068/commits/2140283ee1f7fc02016a09d2b4e08f26a2758c07)

**Review by okynos — APPROVED** [review]

**Pull request merged**